site/app/views/forum/ForumThreadView.php

```diff
@@ -608,7 +600,6 @@ public function displayThreadList($threads, $filtering, &$activeThreadAnnounceme
 
 						//fix legacy code
 						$titleDisplay = html_entity_decode($thread['title'], ENT_QUOTES | ENT_HTML5, 'UTF-8');
-						$first_post_content = html_entity_decode($first_post['content'], ENT_QUOTES | ENT_HTML5, 'UTF-8');
```

Comment:
This should be left in. Discussion can be had if there is a need to support the initial version.

Reply:
If we're leaving in dead code, it should probably be commented out and with a comment as to why it was left there. Right now, this line is immediately overwritten by the next one (as the next one uses `$first_post['content']` and not `$first_post_content`).